src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px;
+
+  &__img-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    margin-top: 40px;
+    display: block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
```

Comment:
set either margin top or bottom, don't mix them up.
it could lead to margin collapce